docs/_documentations/mdt-che-installinfo.md

```diff
@@ -1,62 +1,47 @@
 ---
 layout: docs
-title: Installing Codewind on the Cloud
-description: Installing Codewind on the Cloud
-keywords: build, deploy, IBM Cloud Private, install, installing, installation, chart, Helm, develop, cloud, public cloud, services, command line, cli, command, start, stop, update, open, delete, options, operation, devops, OpenShift, OKD
+title: Installing Codewind for Eclipse Che
+description: Installing Codewind for Eclipse Che
+keywords: build, deploy, install, installing, installation, chart, Helm, develop, cloud, public cloud, services, command line, cli, command, start, stop, update, open, delete, options, operation, devops, OpenShift, OKD
 duration: 1 minute
 permalink: mdt-che-installinfo
 type: document
-order: 20
-parent: root
 ---
 
 # Installing Codewind for Eclipse Che
+Install Che to use with Codewind or use Codewind with an existing Che installation.
 
-Installing Codewind for Eclipse Che comprises the following steps:
-- [Installing Codewind for Eclipse Che](#installing-codewind-for-eclipse-che)
-  - [Prerequisites](#prerequisites)
-- [Setting up Che](#setting-up-che)
-  - [Installing Che with chectl](#installing-che-with-chectl)
-    - [Updating an existing Che install](#updating-an-existing-che-install)
-    - [Enabling privileged and root containers to run](#enabling-privileged-and-root-containers-to-run)
-
-## Prerequisites
-1. Ensure PersistentVolumes (PV) are set up and support both `ReadWriteOnce` and `ReadWriteMany` and have minimum 1Gi storage.
+### Prerequisites
+1. Set up PersistentVolumes (PVs) that support both `ReadWriteOnce` (RWO) and `ReadWriteMany` (RWX) access modes and have a minimum of 1 Gi storage.
    - One volume is required for Che, and two volumes are required for each Codewind workspace.
-   - For NFS, set 777 permissions for the exported folders and ownership of `nobody:nogroup`.
-   - Because Codewind uses RWX (ReadWriteMany) volumes to provide persistent storage, you need to use NFS for storage on OpenShift 4.
+   - For Network File System (NFS), set 777 permissions for the exported folders and ownership of `nobody:nogroup`.
```

Comment:
Maybe update to `Network File System (NFS) shares`? eg. https://kubernetes.io/docs/concepts/storage/volumes/#nfs